### PR TITLE
DTSPO-15799: Refactor pipeline to run sequentially on all enviroments

### PR DESCRIPTION
**Pull request by danielwilsonkainos**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSPO-15799

### Change description ###

- Refactor pipeline to run sequentially on all enviroments: prevents changes which break just one env being merged to master which previously would be missed as PR checks only ran against sbox
- Stages can still be picked individually on manual run by using the 'Stages to run' section on pipeline

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: azure-pipelines-environments.yaml
Comment (danielwilsonkainos) [inline azure-pipelines-environments.yaml:1]: This file isn't needed, its a leftover from when i was trialing something early on. It should be removed

**Comment by NickAzureDevops** [comment]
Those warning on the pipeline on branch are the same like on main branch. checked all of them too

**Review by NickAzureDevops — APPROVED** [review]

File: azure-pipelines-environments.yaml
Comment (danielwilsonkainos) [reply azure-pipelines-environments.yaml:1]: the file is still here

**Review by thomast1906 — APPROVED** [review]
@NickAzureDevops has mentioned its a direct copy from the recent cft pipeline work.

**Pull request merged**